frontend/src/components/ProductTable.js

```diff
@@ -0,0 +1,42 @@
+import {getProducts} from "../services/ProductApiService";
+import {useEffect, useState} from "react";
+import * as React from "react";
+
+export default function () {
+    const [productData, setProductData] = useState([])
```

Comment:
Data is super unspecific, maybe we can find a better name? Something indicating that this is an array would be nice